src/client/src/tscSQLParser.c

```diff
@@ -2277,10 +2270,6 @@ int32_t validateSelectNodeList(SSqlCmd* pCmd, SQueryInfo* pQueryInfo, SArray* pS
        hasDistinct = (pItem->distinct == true);
        distIdx     =  hasDistinct ? i : -1;
     }
-    if(pItem->aliasName != NULL && validateColumnName(pItem->aliasName) != TSDB_CODE_SUCCESS){
```

Comment:
Relax restriction on aliasName, which is allowed to a sequence of words containing capital letters or Chinese characters.